datasetComparison/src/main/scala/za/co/absa/hermes/datasetComparison/cliUtils/DataframeOptions.scala

```diff
@@ -30,20 +32,51 @@ case class DataframeOptions(format: String, options: Map[String, String], path:
     if (format == "jdbc") dfReader.load()
     else dfReader.load(path)
 
-  private def save[T](dfWriter: DataFrameWriter[T], extraPath: String = ""): Unit =
+  private def save[T](dfWriter: DataFrameWriter[T], endPath: String = ""): Unit =
     if (format == "jdbc") dfWriter.save()
-    else dfWriter.save(s"$path$extraPath")
+    else dfWriter.save(endPath)
+
+  def getUniqueFilePath(extraPath: String, conf: Configuration): String = {
```

Comment:
This function with very little modification could be a pure function with usability in other parts of the project. Might be worth it to "publish" it that way. 🤔

Reply:
Thought of it. This is method has another flavour somewhere else in the code. I will see